storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/serialization/RemoteLogMetadataSerde.java

```diff
@@ -107,4 +110,13 @@ private RemoteLogMetadataTransform remoteLogMetadataTransform(short apiKey) {
 
         return metadataTransform;
     }
+
+    public static class RemoteLogMetadataFormatter implements MessageFormatter {
+        private final RemoteLogMetadataSerde remoteLogMetadataSerde = new RemoteLogMetadataSerde();
+
+        @Override
+        public void writeTo(ConsumerRecord<byte[], byte[]> consumerRecord, PrintStream output) {
+            output.println(remoteLogMetadataSerde.deserialize(consumerRecord.value()).toString());
```

Comment:
Please add a comment here that `key` for a `RemoteLogMetadataRecord` is null.

Reply:
Done

Reply:
It is good to include offset and partition, which will be helpful for debugging.

```
output.printf("offset: %d, partition: %d, value: %s%n", record.offset(), record.partition(), remoteLogMetadataSerde.deserialize(consumerRecord.value()).toString());
```

Reply:
Makes sense, done